cli/sync.go

```diff
@@ -64,6 +67,9 @@ var syncWaitCmd = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
+		sigChan := make(chan os.Signal, 2)
+		signal.Notify(sigChan, syscall.SIGTERM, syscall.SIGINT)
```

Comment:
ReqContext already creates context with that - selecting on `ctx.Done()` below should be enough

Reply:
Thank you. Done.